OneBlink.SDK/FormsAppsClient.cs

```diff
@@ -10,7 +10,7 @@ namespace OneBlink.SDK
 {
     public class FormsAppsClient
     {
-        OneBlinkApiClient oneBlinkApiClient;
+        private readonly OneBlinkApiClient oneBlinkApiClient;
```

Comment:
This is also a breaking change. I'm ok with making it readonly, but making it private could break someones code.

Reply:
In C# this is already private implicitly.